colossus/src/main/scala/colossus/protocols/http/HttpServerCodec.scala

```diff
@@ -88,7 +88,6 @@ object StreamedResponseBuilder extends HttpResponseDataReaderBuilder {
     builder.sizeHint(100)
     appendHeaderBytes(response, builder)
     builder append NEWLINE
-    builder append NEWLINE
```

Comment:
This was the heartbreaker...broke streamed http..:).  The tests I wrote in the StreamingHttpresonseParserSpec verify that this is actually working now